crates/rpc/src/jsonrpc/websocket/data.rs

```diff
@@ -211,6 +213,8 @@ impl serde::Serialize for BlockHeader {
         map.serialize_entry("timestamp", &timestamp)?;
         map.serialize_entry("eth_l1_gas_price", &eth_l1_gas_price)?;
         map.serialize_entry("strk_l1_gas_price", &strk_l1_gas_price)?;
+        map.serialize_entry("eth_l1_data_gas_price", &eth_l1_data_gas_price)?;
+        map.serialize_entry("strk_l1_data_gas_price", &strk_l1_data_gas_price)?;
```

Comment:
These new fields should be added to `pathfinder_ws.json`. (And a note added to changelog that we're no longer 100% compatible.)